cx2t/src/claw/wani/transformation/sca/Parallelize.java

```diff
@@ -377,21 +384,25 @@ public void transform(XcodeProgram xcodeml, Translator translator,
   private void transformForGPU(XcodeProgram xcodeml)
       throws IllegalTransformationException
   {
+    CompilerDirective currentDirective = Configuration.get().getCurrentDirective();
+    OpenMpConfiguration openMpConfiguration = Configuration.get().openMP();
+    OpenAccConfiguration openAccConfiguration = Configuration.get().openACC();
+
     // TODO nodep should be passed in another way.
     int collapse = Directive.generateLoopSeq(xcodeml, _fctDef,
         CompilerDirective.CLAW.getPrefix() + " nodep");
 
     if(!Body.isEmpty(_fctDef.body())) {
-      /* Create a nested loop with the new defined dimensions and wrap it around
-       * the whole subroutine's body. This is for the moment a really naive
-       * transformation idea but it is our start point.
-       * Use the first over clause to create it. */
+      // Create a nested loop with the new defined dimensions and wrap it around
```

Comment:
I have done this, because it was annoying when I wanted to do a block comment to test something

Reply:
You can also do it on your side and not commit the change. I would rather like block comment at the end